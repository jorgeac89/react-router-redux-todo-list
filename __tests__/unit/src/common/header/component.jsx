import React from 'react';
import { shallow } from 'enzyme';
import Header from 'common/header';

describe('Header component', () => {
  const HeaderComponent = Header.component;

  it('renders correctly', () => {
    expect(shallow(<HeaderComponent />)).toMatchSnapshot();
  });
});
