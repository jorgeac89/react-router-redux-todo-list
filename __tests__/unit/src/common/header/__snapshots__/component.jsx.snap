// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders correctly 1`] = `
<div
  className="container-fluid bg-dark text-light mb-4"
>
  <div
    className="row align-items-center justify-content-around text-center"
  >
    <div
      className="col-lg-6 text-lg-left"
    >
      <h1
        className="font-weight-light"
      >
        React Router Redux Todo List
      </h1>
      <p
        className="lead"
      >
        Example of a web application developed with 
        <strong>
          React
        </strong>
        , 
        <strong>
          React Router
        </strong>
         and 
        <strong>
          Redux
        </strong>
        .
      </p>
    </div>
    <div
      className="col-lg-4 text-lg-right"
    >
      <img
        alt="logo"
        className="img-fluid mb-3 my-lg-3"
        src="undefined/images/logo.webp"
      />
    </div>
  </div>
</div>
`;
